go-continuous-fuzz-chart/templates/rolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.rbac.create -}}
```

Comment:
This should be unconditional to ensure the svcaccount has the correct rolebinding